common/input/web-payment/index.js

```diff
@@ -33,7 +33,7 @@ function toggleLogVisibility() {
 function initPaymentRequest() {
   const request = new PaymentRequest(
       [{
-        supportedMethods: ['basic-card'],
+        supportedMethods: ['https://bobbucks.dev/pay'],
```

Comment:
```suggestion
        supportedMethods: 'https://bobbucks.dev/pay',
```

Using an array has been deprecated a while ago. Please use a string instead.